docs/modules/ROOT/pages/ios_task_scheduling.adoc

```diff
@@ -0,0 +1,33 @@
+= Task Scheduling
+
+Background tasks are scheduled based on the app's current context, and that context is based on a combination of factors. 
+These factors include:
+
+* Is the app backgrounded or in the foreground?
+* Is WiFi available?
+* Is the device in Low Power Mode?
+* Is the device connected to external power?
+* Has the photo library has changed?
+
+== Tasks 
+
+Currently, two tasks are available:
+
+* *Instant photo uploads:* This task is triggered when:
+** The photo library has new assets
+** The user is connected to a WiFi network
+* *Bookmark update task:* This task is triggered by the background fetch event scheduled by iOS
+
+=== Instant Photo Upload
+
+There are some things to be aware of, regarding Instant Photo Upload.
+
+* Photo and video upload can be enabled and disabled separately.
+* Before instant media upload can begin, the account and upload path have to be selected
```

Comment:
why is it "instant media upload" here, but the heading... is "Instant Photo Upload"

Reply:
Good point. Well spotted.